### ISSUE TITLE
ZIO

### ISSUE THREAD

**Issue by solarmosaic-kflorence** [fosskers/scalaz-and-cats#13]
First off, thank you for providing this resource, it is very helpful. Now that [https://zio.dev/](ZIO) has emerged from ScalaZ, I wonder if this analysis can be updated to include it?

**Comment by fosskers** [comment]
Yes that would be a good thing to do, considering that people still reference this. Would you (or someone you know) be interested in helping with that?

**Comment by solarmosaic-kflorence** [comment]
@fosskers I unfortunately don't have experience with ZIO either :)

**Comment by fosskers** [comment]
@jdegoes is there anybody available who could dive into such a task?
